js/interface.js

```diff
@@ -23,6 +23,15 @@ var DynamicLists = (function() {
       ]
     }
   ];
+  var logicOptions = [
```

Comment:
I don't understand what `logicOptions` is and why it only has these 6 logic operators when we have a lot more. It needs to be clearer.

Reply:
@sofiiakvasnevska This has not been resolved.